<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintView</class>
 <widget class="QWidget" name="PrintView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>792</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">&lt;span style=&quot; font-size:18pt;&quot;&gt;Print&lt;/span&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="copiesBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Copies</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Copies:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="copiesSpin">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Start on position:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="copiesStartSpin">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>on 1st page</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="copiesDescriptionLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>330</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string notr="true">(Will print a total of xxxx items on nnn pages.)        </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Print options</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="printOutlinesCheck">
            <property name="text">
             <string>print outlines</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="printCropMarksCheck">
            <property name="text">
             <string>print crop marks</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="printReverseCheck">
            <property name="text">
             <string>print in reverse (i.e. a mirror image)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="printButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">text-align:left; padding:3px;</string>
       </property>
       <property name="text">
        <string>Print</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/flat/32x32/glabels-print.svg</normaloff>:/icons/flat/32x32/glabels-print.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="glabels::Preview" name="preview" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>520</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="bottomMargin">
        <number>16</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Page</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="pageSpin">
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>of</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nPagesLabel">
         <property name="text">
          <string>nn</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glabels::Preview</class>
   <extends>QWidget</extends>
   <header>Preview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>copiesStartSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PrintView</receiver>
   <slot>onFormChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copiesSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PrintView</receiver>
   <slot>onFormChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printOutlinesCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>PrintView</receiver>
   <slot>onFormChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printCropMarksCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>PrintView</receiver>
   <slot>onFormChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printReverseCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>PrintView</receiver>
   <slot>onFormChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PrintView</receiver>
   <slot>onFormChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>745</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>532</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printButton</sender>
   <signal>clicked()</signal>
   <receiver>PrintView</receiver>
   <slot>onPrintButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPrintButtonClicked()</slot>
  <slot>onFormChanged()</slot>
  <slot>onPrinterPropertiesButtonClicked()</slot>
 </slots>
</ui>
